hermes-frontend/build.gradle

```diff
@@ -14,7 +14,7 @@ dependencies {
     api group: 'org.springframework.boot', name: 'spring-boot-starter', version: versions.spring
     api group: 'io.undertow', name: 'undertow-core', version: versions.undertow
     // Did not update that as we're trying to abandon buffers
-    api(group: 'net.openhft', name: 'chronicle-map', version: '3.22.9') {
+    api(group: 'net.openhft', name: 'chronicle-map', version: '3.25ea6') {
```

Comment:
Newest: [3.27ea0](https://mvnrepository.com/artifact/net.openhft/chronicle-map/3.27ea0)

Reply:
I know, but in my opinion since we are closer to remove this dependency from hermes we do not need the newest version, also I would not want to introduce any changes in apis, so i picked the lowest possible minor version of this dependency